R/utils.R

```diff
@@ -1,6 +1,6 @@
 find_resource <- function(template, file = 'template.tex') {
-  res <- system.file(
-    "rmarkdown", "templates", template, "resources", file, package = "rticles"
+  res <- pkg_file(
+    "rmarkdown", "templates", template, "resources", file
   )
```

Comment:
Since it fits one line (less than 80 chars), I'd merge the three lines:

```suggestion
  res <- pkg_file("rmarkdown", "templates", template, "resources", file)
```